e2e/testcases/managed_resources_test.go

```diff
@@ -708,7 +689,7 @@ func TestDriftRemoveApplySetPartOfLabel(t *testing.T) {
 	nt.Must(nt.Watcher.WatchObject(kinds.Namespace(), namespace, "", testwatcher.WatchPredicates(
 		testpredicates.HasLabel(metadata.ApplySetPartOfLabel, rootSync1ApplySetID),
 		testpredicates.HasAnnotation("season", "summer"),
-	), testwatcher.WatchTimeout(10*time.Second)))
+	), testwatcher.WatchTimeout(driftDetectionTimeout)))
```

Comment:
nit: This is reducing the timeout and it's not clear that it's intentional. Will this make the test more flaky with a shorter timeout?